src/app/caesar-portal/group-area/group-area.component.scss

```diff
@@ -0,0 +1,14 @@
+.group-wrapper {
+	height: 100%;
+}
+
+.group-area {
+	min-height: calc(100% - 200px);
+	border: 1px solid #ccc;
```

Comment:
format your styles, + all colors should be moved to variables so you can reuse them